bin/install-core

```diff
@@ -120,13 +124,30 @@ mkdir -p "$DOWNLOAD_DIR"
 
 # TODO: uncomment the downloads when done testing other things
 # Download Bitcoin core checksums and signatures in background
-# TODO: these are messy make them legible.
-( cd "$DOWNLOAD_DIR"; #wget --tries=1 --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/en/download/ ||
-wget --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin https://bitcoincore.org/en/download/; ) & sig_dl=$!
+(
+    cd "$DOWNLOAD_DIR"
+    wget --tries=1 --no-host-directories --cut-dirs=1 --recursive \
+        #--level=1 --accept=SHA256SUMS* --include-directories=bin \
+        #"https://${BITCOIN_CORE_SOURCE_DOMAIN[0]}/en/download/" ||
+    wget --no-host-directories --cut-dirs=1 --recursive \
+        --level=1 --accept=SHA256SUMS* --include-directories=bin \
+        "https://${BITCOIN_CORE_SOURCE_DOMAIN[1]}/en/download/"
+) & sig_dl=$!
 # TODO FIXME uncomment the onion service when bitcoincore.org fixes their service.
 # Download Bitcoin Core in background to the destination directory
-( sleep 1; while pgrep wget &>/dev/null; do sleep 1; done; cd "$DOWNLOAD_DIR"; #wget --continue --tries=1 --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=x86_64-linux-gnu.tar.gz --include-directories=bin http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/en/download/ ||
-wget --continue --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=x86_64-linux-gnu.tar.gz --include-directories=bin https://bitcoincore.org/en/download/; ) & core_download=$!
+(
+    sleep 1
+    while pgrep wget &>/dev/null; do
+        sleep 1
+    done
+    cd "$DOWNLOAD_DIR"
+    #wget --continue --tries=1 --no-host-directories --cut-dirs=1 --recursive \
+        #--level=1 --accept=x86_64-linux-gnu.tar.gz --include-directories=bin \
+        #"http://${BITCOIN_CORE_SOURCE_DOMAIN[0]}/en/download/" ||
+    wget --continue --no-host-directories --cut-dirs=1 --recursive \
+        --level=1 --accept=x86_64-linux-gnu.tar.gz --include-directories=bin \
+        https://${BITCOIN_CORE_SOURCE_DOMAIN[1]}/en/download/
+) & core_download=$!
```

Comment:
same comments from signatures and checksums apply here.

Reply:
Good idea. Fixed in f59ecac569ca591c342a954f81823cea1f080a63.